src/metabase/models/pulse_channel.clj

```diff
@@ -202,22 +210,6 @@
   [_pulse-channel]
   [(serdes/hydrated-hash :pulse) :channel_type :details :created_at])
 
-(defn will-delete-recipient
```

Comment:
Similarly, this was only used in one place, `metabase.models.pulse-channel-recipient`, so I moved it there, to eliminate dynamic resolution and a false "unused var" warning.